packages/analytics-js/src/state/slices/loadOptions.ts

```diff
@@ -44,9 +44,12 @@ const defaultLoadOptions: LoadOptions = {
   },
   sendAdblockPageOptions: {},
   useServerSideCookies: false,
-  trackPageLifecycle: {
+  autoTrack: {
```

Comment:
For auto-tracking options, we can omit the default values as they don't go through the normalization.

Reply:
Addressed